.env.test

```diff
@@ -1,6 +1,8 @@
 DEBUG=true
 SECRET_KEY='tests'
 DATABASE_URL='postgres://localhost/opencraft'
+INSTANCE_MYSQL_URL='mysql://root@localhost'
+INSTANCE_MONGO_URL='mongodb://localhost'
```

Comment:
They are technically integration tests, but in terms how easy and quick they are to run they are much closer to the existing unit tests than the existing integration tests:

- They run in well under a second, vs several minutes for the integration tests
- No need for any kind of credentials (openstack, etc)
- No need for any manual setup when using vagrant

Installing the databases on the instance VM is just for convenience. I assume that the instance manager's own postgres database is not running on the same VM in production either.

If we add new instance types with other databases in the future, the setup can easily be automated with vagrant. This automation should keep things from getting unwieldy.